Libplanet.Tests/Net/SwarmTest.cs

```diff
@@ -249,12 +249,12 @@ public async Task BroadcastBlockToReconnectedPeer()
                 Assert.Contains(swarmB.AsPeer, seed.Peers);
                 Assert.Contains(seed.AsPeer, swarmB.Peers);
 
-                seed.BroadcastBlocks(new[] { chainWithBlocks.Last() });
+                seed.BroadcastBlocks(new[] { chainWithBlocks[-1] });
```

Comment:
Does it differ from `chainWithBlocks.Tip`?

Reply:
Fixed